Pipfile

```diff
@@ -5,17 +5,17 @@ name = "pypi"
 
 [packages]
 checkov = "==2.2.221"
```

Comment:
Why do we lock checkov?

Reply:
it is automatically updated, when we release a new `checkov` version 🙂